src/main/webapp/js/moreInfo-loader.js

```diff
@@ -0,0 +1,25 @@
+// Get the modal
+const modal = document.getElementById("myModal");
+
+// Get the button that opens the modal
+const btn = document.getElementById("myBtn");
+
+// Get the <span> element that closes the modal
+const span = document.getElementsByClassName("close")[0];
+
+// When the user clicks the button, open the modal
+btn.onclick = function() {
```

Comment:
This button is kind of hard to see for the user in my opinion(but maybe you are purposely hiding it)

Reply:
Yea actually I wanted to hide it, also I thought since the "help" button for most webpages is at the bottom right corner so intuitively most likely it should be ok, but I've enlarged it :)